tests/glommio_ct.rs

```diff
@@ -1,8 +1,8 @@
 #![ cfg( feature = "glommio" ) ]
 
-// ✔ pass a     ThreadPool  to a function that takes exec: `impl SpawnHandle`
-// ✔ pass a Arc<ThreadPool> to a function that takes exec: `impl SpawnHandle`
-// ✔ pass a    &ThreadPool  to a function that takes exec: `&dyn SpawnHandle`
+// ✔ pass a     GlommioCt  to a function that takes exec: `impl SpawnHandle`
```

Comment:
Could you take the file test/tokio_ct.rs instead of the one from `ThreadPool` as a basis? This one doesn't test `LocalSpawn`  and  `LocalSpawnHandle`, as a threadpool does not implement these traits, but for glommio we want them tested.

Reply:
Ok, now I see what happens if a user instantiates LocalExecutor twice in the same thread. If the memory limit in /etc/security/limits.conf is not high enough, it will report OutOfMemory error
https://github.com/DataDog/glommio/issues/1